executor-service/src/main/java/executor/service/App.java

```diff
@@ -1,13 +1,16 @@
 package executor.service;
 
 import executor.service.facade.parallel.ParallelFlowExecutorService;
-import executor.service.factory.di.ApplicationContext;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@SpringBootApplication
 public class App {
 
     public static void main(String[] args) {
-        ApplicationContext applicationContext = new ApplicationContext();
-        ParallelFlowExecutorService executorService = applicationContext.getComponent(ParallelFlowExecutorService.class);
+        ConfigurableApplicationContext applicationContext = SpringApplication.run(App.class);
```

Comment:
if you implement CommandLineRunner, you can only leave
```
    public static void main(String[] args) {
        SpringApplication.run(App.class, args);
    }
```

Reply:
done it, thanks!